rcaracaus/pledge-vegan-node
# 43-get-resource (1)[1]

Steps to test

1. Go to localhost:3000
2. Check every page loads correctly and go into every possible sub-url
